veps/sig-compute/pcihole64-proposal.md

```diff
@@ -0,0 +1,323 @@
+# VEP #33: Allow to disable the 64-bit PCI hole
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone /
+release*.
+
+- [ ] (R) Enhancement issue created, which links to VEP dir
+  in [kubevirt/enhancements] (not the initial VEP PR)
+
+## Overview
+
+<!--
+Provide a brief overview of the topic
+-->
+
+This proposal focuses on allowing to disable the 64-bit PCI hole in
```

Comment:
Feel free to ignore, but could you please include a brief explanation about what 64-bit PCI hole is? Hopefully this will help to understand a bit better the importance of having a way to disable it.